src/app/package-json.js

```diff
@@ -168,13 +168,6 @@ const packageJSON = (current, context) => {
     'author': context.author,
     'license': context.licenseName,
 
-    'browserify': {
-      transform: [
-        'browserify-shim',
-        'browserify-versionify'
```

Comment:
I guess. Will browserify merge the `-t` transforms with those in the `package.json`?